### DISCUSSION TITLE
Add USB Zero delay encoder compatibility

### DISCUSSION THREAD

**Discussion by Deepack57** [cathery/sys-con#205] [Controller requests]
Hello,

First at all, great job with Sys-con!

I'm actually building a Arcade Machine, where I want to play with my controller (one joystick & 6 button) to my switch. Already tried to simply connect the interface to my switch, and no surprise, it didn't work.

Maybe could you make this compatibily?

(And sorry for the pull request, my bad)

Here is the screenshot 

![2020080800521400-DB1426D1DFD034027CECDE9C2DD914B8](https://user-images.githubusercontent.com/68164894/89695062-02a20500-d913-11ea-8206-64694c4081be.jpg)

**Comment by cathery** [comment]
Support for HID devices will come eventually

**Comment by Deepack57** [comment]
Oh, nice! Do you have an approximate date? :)

**Comment by cathery** [comment]
very far away
